gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -167,12 +167,29 @@ function LANG.GetUnsafeLanguageTable()
 end
 
 ---
--- Returns a translated @{string} text (if possible) directly from the language table
--- @param string name string key identifier for the translated @{string}
--- @return nil|string
+-- Returns the reference to a language table if it exists
+-- @param string name string key identifier for the language
+-- @return nil|table
+-- @realm client
+function LANG.GetUnsafeNamed(lang_name)
+	lang_name = lang_name and string.lower(lang_name)
+
+	return LANG.Strings[lang_name]
+end
+
+---
+-- Returns the reference to a language table if it exists, creates a new language if it did not exist
+-- @param string name string key identifier for the language
+-- @return nil|table
 -- @realm client
-function LANG.GetUnsafeNamed(name)
-	return LANG.Strings[name]
+function LANG.GetOrCreateLanguage(lang_name)
+	lang_name = lang_name and string.lower(lang_name)
```

Comment:
or?

Reply:
I copied it from somewhere else in this file. I think it is to make sure the language stays nil if you forget to set `lang_name`

Reply:
Then you should add `or nil` as already talked about. But following the next try of indexing nil seems to be wrong imo

Reply:
I see your point. But it was like this forever. And I don't want to change it without knowing why it was done like this

Reply:
It was done like this because the whole lang thing was a unstable construct you already mentioned on your own. As you did in the other functions, I suggest creating an own error to prevent some weird nil error. Just a suggestions as it improves the development and prevent issues. Additional checks here shouldn‘t lower the performance. If you don‘t think so, feel free to resolve this :)